ruleset.xml

```diff
@@ -5,5 +5,7 @@
 	<file>example.php</file>
 	<arg name="report" value="full"/>
 	<arg value="sp"/>
-	<rule ref="WordPress"/>
```

Comment:
Oh and that's quite apart still from the buggyness of some of the VIP rules, such as https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/issues/186 which will break our build for sure.